### PR TITLE
Give an example for `fields` in the attribute's section too.

### PR DESCRIPTION
**Pull request by webmat**

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -212,7 +201,16 @@ Whether results should be sorted or not
...
-Array of fields to copy from old event (found via elasticsearch) into new event
+An array of fields to copy from the old event (found via elasticsearch) into the
+new event, currently being processed.
+
+In the following example, the value of `@timestamp` on the event found via
+elasticsearch is copied to the current event's `started` field:
+
+[source,ruby]
+--------------------------------------------------
+fields => { "@timestamp" => "started" }
```
[the remark this answers is not in the document]
Comment (webmat) [reply docs/index.asciidoc]: Yes, I can do that. Good idea

File: docs/index.asciidoc [hunk 2]
```diff
@@ -204,12 +204,16 @@ Whether results should be sorted or not
...
+`started` and `start_id` fields, respectively:
 
 [source,ruby]
 --------------------------------------------------
-fields => { "@timestamp" => "started" }
+fields => {
+  "@timestamp" => "started"
+  "event_id" => "start_id"
+}
 --------------------------------------------------
```
Comment (webmat) [inline docs/index.asciidoc]: @robbavey How about this?

Comment (robbavey) [reply docs/index.asciidoc]: @webmat Yes, I think that is fine

Comment (robbavey) [reply docs/index.asciidoc]: The only weird thing is that the documentation states `array` and that is a `hash`, but I guess that is down to the old array of tuples syntax?

Comment (webmat) [reply docs/index.asciidoc]: Yes. It's a question I'll bring up at the next sync.

I'm wondering what the process is to move from the deprecated array to hash, for arrays that work like this attribute (a series of tuples). Unfortunately just making this a hash doesn't work. Hash attributes don't support arrays of tuples, so it looks like this requires a breaking change...

Comment (webmat) [reply docs/index.asciidoc]: At least arrays support the hash notation... Although this causes never-ending confusion :-)

**Pull request closed without merging**